examples/gen.rs

```diff
@@ -325,7 +329,7 @@ fn write_types(enums: &[EnumType], structs: &[Rc<StructType>]) {
             println!("    pub {}: {},", field.field_name, field_type);
         }
 
-        println!("}}\n");
+        println!("}}");
```

Comment:
Should not be changed, althought we run `cargo fmt` afterwards.

Reply:
:heavy_check_mark: